flink-filesystems/flink-azure-fs-hadoop/pom.xml

```diff
@@ -77,4 +77,53 @@ under the License.
 
 	</dependencies>
 
+	<build>
+		<plugins>
+			<!-- Relocate all Azure related classes -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>shade-flink</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<shadeTestJar>false</shadeTestJar>
+							<artifactSet>
+								<includes>
+									<include>*:*</include>
+								</includes>
+							</artifactSet>
+							<relocations>
+								<relocation>
+									<pattern>org.apache.hadoop</pattern>
+									<shadedPattern>org.apache.flink.fs.shaded.hadoop3.org.apache.hadoop</shadedPattern>
```

Comment:
I guess it depends on `hadoop.version` whether it's `hadoop3` or not. Thus, I would simply remove `hadoop3` here.

Reply:
yeah agree, makes sense. Will update